cfg/common_test.go

```diff
@@ -374,6 +393,13 @@ backendsByDC:
           - name: "cluster2"
             backends:
             - "http://10.290.202.32:8080"
+    - name: "dc3"
+      clusters:
+          - name: "cluster1"
+            protocolBackends:
+            - http: "http://10.291.202.31:8080"
+            - http: "http://10.291.197.91:8080"
+              grpc: "10.291.197.91:7070"
```

Comment:
it's minor, but you can simplify things by:

```
protocolBackends:
- "http://10.291.202.31:8080"
- "http://10.291.197.91:8080"
  "grpc://10.291.197.91:7070"
```

net/url pkg actually supports parsing it: https://pkg.go.dev/net/url@go1.18.4#Parse

Reply:
I have difficulty understanding the for `protocolBackend`. Do you mean it can be simplified to list of strings? In that case it would be like:
```protocolBackends:
- ["http://10.291.202.31:8080"]
- ["http://10.291.197.91:8080",
  "grpc://10.291.197.91:7070"]```

Which is not desired IMO. However, after implementing gRPC for all of the calls, it can be simplified to one string in which url scheme determines the backend type.